connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -648,7 +636,6 @@ private static InputStream getInputStream(final CloseableHttpResponse response)
             @Override
             public void close() throws IOException {
                 response.close();
-                super.close();
             }
```

Comment:
If the application consumes the entity, by the time response.close() runs the connection has already released to the pool, so response.close() does nothing.

If the application does NOT consume the entity, there is no other choice but closing the connection - reusing it is not possible in that case.